kinto_client/__init__.py

```diff
@@ -279,9 +279,15 @@ def create_collection(self, collection=None, bucket=None,
                                               safe=safe)
         headers = DO_NOT_OVERWRITE if safe else None
         endpoint = self._get_endpoint('collection', bucket, collection)
-        resp, _ = self.session.request('put', endpoint, data=data,
-                                       permissions=permissions,
-                                       headers=headers)
+        try:
+            resp, _ = self.session.request('put', endpoint, data=data,
+                                           permissions=permissions,
+                                           headers=headers)
+        except KintoException as e:
+            if e.response.status_code == 403:
+                msg = "Unauthorized. Please check that the bucket exists."
```

Comment:
that the bucket exists and that you have the permission to create or write.

?